fusillade/clouddirectory.py

```diff
@@ -15,8 +15,33 @@
 from urllib.parse import quote, unquote
 import os
 
-ad = boto3.client("clouddirectory")  # TODO move to config.py
 project_arn = "arn:aws:clouddirectory:us-east-1:861229788715:"  # TODO move to config.py
+ad = boto3.client("clouddirectory")
```

Comment:
any significance for calling the `cloud_directiory_client`, `ad`?

Reply:
it was for active directory and cd is too similar to linux. I was going to change it to cd_client